### PR TITLE
Don't use disovery for kafka and gRPC

### PR DESCRIPTION
**Pull request by chbatey**
I don't see it adds any value. In kubernetes we'll likely access
dependencies via a service so a simple config property is fine.

If we agree i'll change the other services too.

### CODE REVIEW

File: docs-source/docs/modules/shopping-microservices/examples/shopping-cart-service-scala/src/main/resources/local-shared.conf [hunk 1]
```diff
@@ -26,24 +26,10 @@ akka.remote.artery.canonical.hostname = "127.0.0.1"
...
-akka.discovery.config.services = {
-  // The Shopping Order service's gRPC endpoint
-  "order-service-grpc" = {
-    endpoints = [
-      {host = "localhost", port = 8301}
-    ]
-  }
-}
+order-service.host = "localhost"
+order-service.port = 8301
```
Comment (patriknw) [inline docs-source/docs/modules/shopping-microservices/examples/shopping-cart-service-scala/src/main/resources/local-shared.conf:34]: also put these in application.conf, even though they will not be used until we deploy

File: docs-source/docs/modules/shopping-microservices/examples/shopping-cart-service-scala/src/main/resources/local-shared.conf [hunk 2]
```diff
@@ -26,24 +26,10 @@ akka.remote.artery.canonical.hostname = "127.0.0.1"
...
-
-akka.discovery.config.services = {
-  // The Kafka broker's bootstrap servers
-  "shopping-kafka-broker" = {
-    endpoints = [
-      {host = "localhost", port = 9092}
-    ]
-  }
-}
+shopping-cart.kafka.bootstrap-servers = "localhost:9092"
```
Comment (patriknw) [inline docs-source/docs/modules/shopping-microservices/examples/shopping-cart-service-scala/src/main/resources/local-shared.conf:29]: also in kafka.conf

**Review by patriknw — COMMENTED** [review]
looking good, seems easier to me

**Comment by chbatey** [comment]
I'll do this for all the services now

**Comment by patriknw** [comment]
sorry, I also removed that file

File: docs-source/docs/modules/shopping-microservices/examples/shopping-cart-service-scala/src/main/resources/local-shared.conf (hunk 1 above)
Comment (patriknw) [reply docs-source/docs/modules/shopping-microservices/examples/shopping-cart-service-scala/src/main/resources/local-shared.conf:33]: I have changed those in other places to be full names: shopping-order-service

**Review by patriknw — APPROVED** [review]
LGTM, after small rename

**Comment by patriknw** [comment]
Refs #79

**Pull request merged**